### PR TITLE
fix(styling) Change main Page element to Panel, due to not using

### LINKED ISSUE #42: View Console bar is not repositioned when closing output views when window is maximized
NotRepositionedViewConsoelBar

### LINKED ISSUE #114: Kaoto UI is not fitting the whole editor with Kaoto UI 0.6.0
image

### PR DESCRIPTION
**Pull request by mmelko**
navigation and header anymore
fix https://github.com/KaotoIO/vscode-kaoto/issues/114 
fix https://github.com/KaotoIO/vscode-kaoto/issues/42

### CODE REVIEW

File: src/components/Visualization.tsx [hunk 1]
```diff
@@ -262,8 +262,8 @@ const Visualization = ({ toggleCatalog }: IVisualization) => {
           data-testid={'react-flow-wrapper'}
           ref={reactFlowWrapper}
           style={{
-            width: window.innerWidth,
-            height: window.innerHeight - 77,
+            width: '100vw',
+            height: 'calc(100vh - 77px )',
```
Comment (apupier) [inline src/components/Visualization.tsx:266]: for another time: might be nice to provide a variable/constant for the 77px to know from where it si coming from

File: src/layout/Dashboard.tsx [hunk 2]
```diff
@@ -98,7 +98,8 @@ const Dashboard = () => {
         isResizable={true}
         panelContent={drawerConsole}
         position="bottom"
-        style={{ maxHeight: window.innerHeight - 28 }}
+        // 29px is the size of bottom Console banner
+        style={{ height: 'calc(100vh - 29px)' }}
```
Comment (apupier) [inline src/layout/Dashboard.tsx:102]: for another time: might be nice to provide a variable/constant for the 29px to know from where it is coming from

**Review by apupier — APPROVED** [review]
working!

image

fixedVSCodeDisplay

File: src/layout/Dashboard.tsx (hunk 2 above)
Comment (mmelko) [reply src/layout/Dashboard.tsx:102]: @apupier yeah, It'd be good to reactor it a little bit and get rid of the magic numbers.

File: src/components/Visualization.tsx (hunk 1 above)
Comment (kahboom) [reply src/components/Visualization.tsx:266]: agree with @apupier on this, there are different parts of the app that could benefit from this, so I'll look into it

**Review by kahboom — APPROVED** [review]

**Pull request merged**